core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -39,8 +39,9 @@ THE SOFTWARE.
       <j:if test="${!c.acceptingTasks}"> <st:nbsp/> (${%suspended})</j:if>
     </d:tag>
     <d:tag name="executor">
+      <j:set var="ae" value="${e.asynchronousExecution}"/>
```

Comment:
How do I stop getting these emails?

Sent from my iPhone

> On 25 Mar 2015, at 15:25, Jesse Glick <notifications@github.com> wrote:
> 
> In core/src/main/resources/lib/hudson/executors.jelly:
> 
> > @@ -39,8 +39,9 @@ THE SOFTWARE.
> >        <j:if test="${!c.acceptingTasks}"> <st:nbsp/> (${%suspended})</j:if>
> >      </d:tag>
> >      <d:tag name="executor">
> > +      <j:set var="ae" value="${e.asynchronousExecution}"/>
> If we capture ae and then it goes idle, there is no harm done that I can see. But yes it would be nice to incorporate into DisplayExecutor.
> 
> —
> Reply to this email directly or view it on GitHub.
>